app/views/layouts/_header.html.erb

```diff
@@ -25,7 +25,7 @@
                 </li>
                 <% if feature_active? :volunteer_ops_list %>
                   <li class="<%= active_if 'volunteer_ops' %>">
-                    <%= link_to "Volunteers", volunteer_ops_path %>
+                    <%= link_to "Volunteers", root_path %>
                   </li>
```

Comment:
Because the tests are using this path. For example

```
  Scenario Outline: Find help with care for elderly
    Given I visit the <page>
    Then I should see an <status> home button in the header
    And I click "Home"
    Then I should be on the home page
    Examples:
      | page                                              | status   |
      | home page                                         | active   |
      | volunteer opportunities page                      | inactive |
      | organisations index page                          | inactive |
      | show page for the organisation named "Age UK"     | inactive |

```
This tests makes sure `Home` button is inactive when Volunteer path is visited. And this test doesn't fail when I make the above change simply because the tests are visiting the `volunteer_ops_path` not the original path which Volunteer button points to. 

Does that make sense?